notebooks/official/CODEOWNERS

```diff
@@ -30,3 +30,4 @@
 /custom/custom_training_tensorboard_profiler.ipynb @itseric
 /workbench/spark/spark_sample_notebook.ipynb @bmiro
 /model-registry/bqml-vertexai-model-registry.ipynb @soheilazangeneh
+
```

Comment:
I think we accidentally deleted your line. Can you add your notebook back?

Reply:
Added it back at bottom.